### PR TITLE
Add table `github_pull_request_review`

### PR DESCRIPTION
**Pull request by jramosf**
This Pull Request adds the table `github_pull_request_review`, in a very similar implementation as the existing `github_pull_request_comment`. See https://docs.github.com/en/graphql/reference/objects#pullrequestreview for details.

# Example query results
```sql
select
  rv.*
from
  github_pull_request r
join
  github_pull_request_review rv
on
  r.repository_full_name = rv.repository_full_name
and
  r.number = rv.number
where
  r.repository_full_name = 'turbot/steampipe-plugin-github'
and
  r.state = 'OPEN';
```

```
+--------------------------------+--------+------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------+--------------------+--------------------------------------------------------------------------------------------------------------+-------------------+-----------------------------------------------------------------------------------------+---------------------------+------------------------------+
| repository_full_name           | number | id         | node_id              | author                                                                                                                                                                        | author_login | author_association | body                                                                                                         | state             | url                                                                                     | submitted_at              | _ctx                         |
+--------------------------------+--------+------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------+--------------------+--------------------------------------------------------------------------------------------------------------+-------------------+-----------------------------------------------------------------------------------------+---------------------------+------------------------------+
| turbot/steampipe-plugin-github | 207    | 1218250696 | PRR_kwDOE8SFYs5InQfI | {"avatar_url":"https://avatars.githubusercontent.com/u/12363488?u=9a975a05cc1d66758ff13d1e0a4fdb5758844696\u0026v=4","login":"cbruno10","url":"https://github.com/cbruno10"}  | cbruno10     | CONTRIBUTOR        |                                                                                                              | COMMENTED         | https://github.com/turbot/steampipe-plugin-github/pull/207#pullrequestreview-1218250696 | 2022-12-14T23:12:23+02:00 | {"connection_name":"github"} |
| turbot/steampipe-plugin-github | 207    | 1139029908 | PRR_kwDOE8SFYs5D5DeU | {"avatar_url":"https://avatars.githubusercontent.com/u/12363488?u=9a975a05cc1d66758ff13d1e0a4fdb5758844696\u0026v=4","login":"cbruno10","url":"https://github.com/cbruno10"}  | cbruno10     | CONTRIBUTOR        |                                                                                                              | COMMENTED         | https://github.com/turbot/steampipe-plugin-github/pull/207#pullrequestreview-1139029908 | 2022-10-12T15:48:25+03:00 | {"connection_name":"github"} |
| turbot/steampipe-plugin-github | 207    | 1138822886 | PRR_kwDOE8SFYs5D4Q7m | {"avatar_url":"https://avatars.githubusercontent.com/u/47312748?u=7137c577081452c42ac35955a3c0c5050f472f12\u0026v=4","login":"misraved","url":"https://github.com/misraved"}  | misraved     | CONTRIBUTOR        |                                                                                                              | COMMENTED         | https://github.com/turbot/steampipe-plugin-github/pull/207#pullrequestreview-1138822886 | 2022-10-12T13:14:38+03:00 | {"connection_name":"github"} |
| turbot/steampipe-plugin-github | 207    | 1138817082 | PRR_kwDOE8SFYs5D4Pg6 | {"avatar_url":"https://avatars.githubusercontent.com/u/47312748?u=7137c577081452c42ac35955a3c0c5050f472f12\u0026v=4","login":"misraved","url":"https://github.com/misraved"}  | misraved     | CONTRIBUTOR        |                                                                                                              | COMMENTED         | https://github.com/turbot/steampipe-plugin-github/pull/207#pullrequestreview-1138817082 | 2022-10-12T13:10:18+03:00 | {"connection_name":"github"} |
| turbot/steampipe-plugin-github | 207    | 1214381851 | PRR_kwDOE8SFYs5IYf8b | {"avatar_url":"https://avatars.githubusercontent.com/u/12363488?u=9a975a05cc1d66758ff13d1e0a4fdb5758844696\u0026v=4","login":"cbruno10","url":"https://github.com/cbruno10"}  | cbruno10     | CONTRIBUTOR        | @aminvielledebatAtBedrock Please see review questions and suggestions, thanks!                               | CHANGES_REQUESTED | https://github.com/turbot/steampipe-plugin-github/pull/207#pullrequestreview-1214381851 | 2022-12-13T00:00:29+02:00 | {"connection_name":"github"} |
| turbot/steampipe-plugin-github | 207    | 1214510448 | PRR_kwDOE8SFYs5IY_Vw | {"avatar_url":"https://avatars.githubusercontent.com/u/6843140?u=2290c0653fe8762b24a457e9393fe72b5e18b8a9\u0026v=4","login":"johnsmyth","url":"https://github.com/johnsmyth"} | johnsmyth    | CONTRIBUTOR        |                                                                                                              | COMMENTED         | https://github.com/turbot/steampipe-plugin-github/pull/207#pullrequestreview-1214510448 | 2022-12-13T01:47:26+02:00 | {"connection_name":"github"} |
| turbot/steampipe-plugin-github | 207    | 1138817305 | PRR_kwDOE8SFYs5D4PkZ | {"avatar_url":"https://avatars.githubusercontent.com/u/47312748?u=7137c577081452c42ac35955a3c0c5050f472f12\u0026v=4","login":"misraved","url":"https://github.com/misraved"}  | misraved     | CONTRIBUTOR        |                                                                                                              | COMMENTED         | https://github.com/turbot/steampipe-plugin-github/pull/207#pullrequestreview-1138817305 | 2022-10-12T13:10:28+03:00 | {"connection_name":"github"} |
| turbot/steampipe-plugin-github | 207    | 1138848815 | PRR_kwDOE8SFYs5D4XQv | {"avatar_url":"https://avatars.githubusercontent.com/u/47312748?u=7137c577081452c42ac35955a3c0c5050f472f12\u0026v=4","login":"misraved","url":"https://github.com/misraved"}  | misraved     | CONTRIBUTOR        |                                                                                                              | COMMENTED         | https://github.com/turbot/steampipe-plugin-github/pull/207#pullrequestreview-1138848815 | 2022-10-12T13:35:07+03:00 | {"connection_name":"github"} |
| turbot/steampipe-plugin-github | 207    | 1138822111 | PRR_kwDOE8SFYs5D4Qvf | {"avatar_url":"https://avatars.githubusercontent.com/u/47312748?u=7137c577081452c42ac35955a3c0c5050f472f12\u0026v=4","login":"misraved","url":"https://github.com/misraved"}  | misraved     | CONTRIBUTOR        |                                                                                                              | COMMENTED         | https://github.com/turbot/steampipe-plugin-github/pull/207#pullrequestreview-1138822111 | 2022-10-12T13:14:03+03:00 | {"connection_name":"github"} |
| turbot/steampipe-plugin-github | 207    | 1138944016 | PRR_kwDOE8SFYs5D4ugQ | {"avatar_url":"https://avatars.githubusercontent.com/u/47312748?u=7137c577081452c42ac35955a3c0c5050f472f12\u0026v=4","login":"misraved","url":"https://github.com/misraved"}  | misraved     | CONTRIBUTOR        | Thanks @aminvielledebatAtBedrock for the contribution.                                                       | COMMENTED         | https://github.com/turbot/steampipe-plugin-github/pull/207#pullrequestreview-1138944016 | 2022-10-12T14:51:05+03:00 | {"connection_name":"github"} |
|                                |        |            |                      |                                                                                                                                                                               |              |                    |                                                                                                              |                   |                                                                                         |                           |                              |
|                                |        |            |                      |                                                                                                                                                                               |              |                    | I have left a few minor comments on the PR. If you have any questions on the feedback, please reach out 😄 . |                   |                                                                                         |                           |                              |
| turbot/steampipe-plugin-github | 207    | 1138814978 | PRR_kwDOE8SFYs5D4PAC | {"avatar_url":"https://avatars.githubusercontent.com/u/47312748?u=7137c577081452c42ac35955a3c0c5050f472f12\u0026v=4","login":"misraved","url":"https://github.com/misraved"}  | misraved     | CONTRIBUTOR        |                                                                                                              | COMMENTED         | https://github.com/turbot/steampipe-plugin-github/pull/207#pullrequestreview-1138814978 | 2022-10-12T13:08:42+03:00 | {"connection_name":"github"} |
| turbot/steampipe-plugin-github | 207    | 1138816795 | PRR_kwDOE8SFYs5D4Pcb | {"avatar_url":"https://avatars.githubusercontent.com/u/47312748?u=7137c577081452c42ac35955a3c0c5050f472f12\u0026v=4","login":"misraved","url":"https://github.com/misraved"}  | misraved     | CONTRIBUTOR        |                                                                                                              | COMMENTED         | https://github.com/turbot/steampipe-plugin-github/pull/207#pullrequestreview-1138816795 | 2022-10-12T13:10:04+03:00 | {"connection_name":"github"} |
+--------------------------------+--------+------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------+--------------------+--------------------------------------------------------------------------------------------------------------+-------------------+-----------------------------------------------------------------------------------------+---------------------------+------------------------------+

```
</details>

See `github_pull_request_review.md` for additional examples.

### CODE REVIEW

**Review by shmuelmor — APPROVED** [review]

**Review by misraved — COMMENTED** [review]
Thanks @jramosf for creating this new table 👍.

Please take a look at the minor review comments 👍.

**Comment by jramosf** [comment]
@misraved Thanks for the quick review, I've done the changes (I've not force-pushed but this can be rebased as a one commit when merged with `main` branch).

File: github/models/pull_request.go [hunk 1]
```diff
@@ -99,6 +99,18 @@ type PullRequest struct {
 	// TimelineItems [pageable]
 }
 
+type PullRequestReview struct {
+	Id                        int                               `graphql:"id: databaseId" json:"id"`
+	NodeId                    string                            `graphql:"nodeId: id" json:"node_id"`
+	Author                    Actor                             `json:"author"`
+	AuthorAssociation         githubv4.CommentAuthorAssociation `json:"author_association"`
+	AuthorCanPushToRepository bool                              `json:"autor_can_push_to_repository"`
```
Comment (graza-io) [inline github/models/pull_request.go:107]:
```suggestion
	AuthorCanPushToRepository bool                              `json:"author_can_push_to_repository"`
```

File: github/table_github_pull_request_review.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+	"github.com/shurcooL/githubv4"
+	"github.com/turbot/steampipe-plugin-github/github/models"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func pullRequestReviewColumns() []*plugin.Column {
+	return []*plugin.Column{
```
[the remark this answers is not in the document]
Comment (jramosf) [reply github/table_github_pull_request_review.go:14]: Thanks, I forgot about that!

**Comment by jramosf** [comment]
Addressed all comments, ready-to-review

**Review by graza-io — APPROVED** [review]
Code looks good, tested working as intended.

**Review by misraved — APPROVED** [review]

**Pull request merged**